src/components/Footer/index.tsx

```diff
@@ -262,15 +261,15 @@ export const Footer: React.FC<FooterType> = props => {
                   <FacebookIcon />
                 </a>
               </div>
-            </Cell>
+            </div>
 
-            <Cell cols={4} colsM={8}>
+            <div className={['cols-6 cols-m-8'].filter(Boolean).join(' ')}>
               <p
                 className={classes.copyright}
               >{`Copyright ${new Date().getFullYear()} Payload CMS, Inc.`}</p>
-            </Cell>
+            </div>
 
-            <Cell cols={2} colsM={8} className={classes.themeCell}>
+            <div className={[classes.themeCell, 'cols-2 cols-m-8'].filter(Boolean).join(' ')}>
```

Comment:
@paulpopus Are we sure this should remain `cols-2` even with the new `column` count?

Reply:
In this particular case it's needed, the component is slightly wider than 1 col, so 2 cols keeps it aligned horizontally with the component above it